internal/engine/engine.go

```diff
@@ -72,6 +72,9 @@ type CraneEngine struct {
 	// the registry crane connects with.
 	Insecure bool
 
+	// ManifestListDigest is the sha256 digest for the manifest list
+	ManifestListDigest string
```

Comment:
Should we have used a digest type here?

Reply:
Meaning cranev1.Hash?

Reply:
Yes, exactly - not sure if that adds value other than maybe validation?

Reply:
We can look into this in the future